Makefile

```diff
@@ -140,7 +140,7 @@ container-tests:  test/container/images/centos7/.uptodate pkg/apis/wksprovider/m
 	go test -count=1 ./test/container/...
 
 integration-tests-container: cmd/wksctl/wksctl cmd/controller/.uptodate
-	IMAGE_TAG=$(IMAGE_TAG) go test -v -timeout 20m ./test/integration/container/...
+	IMAGE_TAG=$(IMAGE_TAG) go test -v -timeout 50m ./test/integration/container/...
```

Comment:
Does it really take this long?

Reply:
Unfortunately it does, although I haven't been able to run it successfully on CI. This needs further investigation. In the meantime, it's fine to keep this to 20 mins.

Reply:
Done